src/experiment/trials/introduction.js

```diff
@@ -0,0 +1,209 @@
+import { initJsPsych } from 'jspsych';
+import htmlKeyboardResponse from '@jspsych/plugin-html-keyboard-response';
+import videoKeyboardResponse from '@jspsych/plugin-video-keyboard-response';
+import { pressKey } from '@jspsych/test-utils';
+import 'jspsych/css/jspsych.css';
+import 'regenerator-runtime/runtime';
+import { mediaAssets } from '../loadassets';
+import { camelCase } from 'lodash';
+
+const jsPsychForURL = initJsPsych();
+const language = jsPsychForURL.data.getURLVariable('language') || 'en';
+const responseModality =
+  jsPsychForURL.data.getURLVariable('responseModality') || 'touch';
+export const enableButtons = responseModality === 'touch' ? true : false;
+let buttonClicked = false;
+
+export const videos =
+  language === 'es'
+    ? {
+        introVideo1: mediaAssets.video[camelCase("Spanish_Honey_Hunt_Intro_1")],
+        introVideo2: mediaAssets.video[camelCase("Spanish_Honey_Hunt_Intro_2")],
+        introVideo3: mediaAssets.video[camelCase("Spanish_Honey_Hunt_Intro_3")],
+        introVideo4: mediaAssets.video[camelCase("Spanish_Honey_Hunt_Intro_4")],
+        introButtonVideo3: mediaAssets.video[camelCase("Spanish_Honey_Hunt_Intro_Button_3")],
```

Comment:
I don't think we should do this as it adds extra work. The file names are already descriptive so we should access them directly. We also don't need the camelCase function, the generateAssetObject function does that under the hood and removes underscores. So it will be mediaAssets.video.spanishHoneyHuntIntro1

Reply:
How should we handle the language parameter?